src/gtest/test_sidechain.cpp

```diff
@@ -1020,6 +1037,107 @@ TEST_F(SidechainTestSuite, CSidechainFromMempoolRetrievesUnconfirmedInformation)
     EXPECT_TRUE(retrievedInfo.mImmatureAmounts.at(-1) == fwdAmount);
 }
 
+TEST_F(SidechainTestSuite, CSidechainBlockUndoVersioning) {
+
```

Comment:
In other UTs involving serialization ops, I have used a `CDataStream` type. Is it reusable here? I'd make these UTs more isolated wrt the environment and prove the

Reply:
Agree, on the other side is also useful testing the specific stream CAutoFile, which is used by the code for reading/writing the undo file